SwitchItem/SwitchItem.module.less

```diff
@@ -6,9 +6,44 @@
 .switchItem {
 	.switchIcon {
 		pointer-events: none;
+	}
+
+	.applySkins({
+		height: @agate-switchitem-height;
```

Comment:
I think it's better to expose `size` prop of `Item` which is the base component of `SwitchItem` instead of overriding this. WDYT?
If someone wants to have a list with Item(size default large) and SwitchItem(100px), the layout would look lack of unity.
I think the Item kind component should have the same default height.

Reply:
I agree. I also added a `size` knob inside SwitchItem sample, which inherits the styles form Item. Do you think is neccessary?

Reply:
👍